Lib/venv/scripts/nt/activate.bat

```diff
@@ -1,13 +1,11 @@
 @echo off
 
-rem This file is UTF-8 encoded, so we need to update the current code page while executing it
-for /f "tokens=2 delims=:" %%a in ('"%SystemRoot%\System32\chcp.com"') do (
-    set "_OLD_CODEPAGE=%%a"
-)
+rem This file is UTF-8 encoded, so we need to update the current code page while executing it.
+for /f %%a in ('%~dp0python.exe -s -S -E -c "import ctypes; print(ctypes.windll.kernel32.GetConsoleOutputCP())"') do (set "_OLD_CODEPAGE=%%a")
```

Comment:
The [-I option](https://docs.python.org/3/using/cmdline.html#id2) (3.4+) implies `-Es` and also removes the user site packages and script directory from `sys.path`. The script directory should be the working directory for a `-c` command. I don't think we need to skip importing the site module via `-S`.